core/domain/email_manager.py

```diff
@@ -154,7 +154,7 @@ def _require_sender_id_is_valid(intent, sender_id):
 
 def _send_email(
         recipient_id, sender_id, intent, email_subject, email_html_body,
-        bcc_admin=False):
+        sender_email=feconf.SYSTEM_EMAIL_ADDRESS, bcc_admin=False):
```

Comment:
Can we force the caller to specify a sender email address, and not provide a default arg?